extensions/amp-story/1.0/page-advancement.js

```diff
@@ -426,14 +432,21 @@ class ManualAdvancement extends AdvancementConfig {
     let shouldHandleEvent = false;
     let tagName;
 
+    if (this.isInScreenSideEdge_(event, this.element_.getLayoutBox())) {
```

Comment:
You still need to make sure the click comes from within an `amp-story-page` though. For example, it could come from the system layer (clicking on the share button?)

Reply:
Ok, I just simplified this so that the screen edge check only comes when clicking on a quiz